articles.tex

```diff
@@ -435,6 +435,14 @@
 Vote Counters are the chairperson of the vote, and two randomly selected members.
 
 \asection{Types of Voting}
+\asubsection{Alternative Vote}
+\asubsubsection{Method of Vote}
+Votes are cast on paper ballots, which provide a means to indicate every possible option in the vote. A ballot is then distributed to each House member eligible to cast a vote and collected in the designated ballot box for a pre-specified length of time. At the end of the voting period, the person chairing the vote collects the ballots, closing the voting period. The Vote Counters then tally the results.
+
+Voters rank the options by placing a '1' by their first choice, a '2' by their second choice, and so on, until they no longer wish to express any further preferences or run out of options.
+
+\asubsubsection{Voting Period} 
+For constitutional modification, candidate selection, and officer removal votes, the voting period must be at least forty-eight (48) hours in length. For any other type of vote, the voting period must be at least twenty-four (24) hours. The minimum length of the voting period may be explicitly lengthed, but never shortened, in the text describing the actual vote.

```

Comment:
> ... in the text describing the actual vote.
Is this clause meant to restrict changes to the voting period to specific to the descriptions of various votes in the constitution, or on the ballots themselves?